ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/ws/ThrowableExceptionMapper.java

```diff
@@ -49,9 +50,20 @@ public Response toResponse(Throwable exception) {
     var acceptable = headers.getAcceptableMediaTypes();
     var status = determineStatus(exception);
 
+    var acceptableLanguages = headers.getAcceptableLanguages();
+    if (acceptableLanguages == null
+        || acceptableLanguages.isEmpty()
+        || (acceptableLanguages.stream().noneMatch(Locale.ENGLISH::equals)
+            && acceptableLanguages.stream().noneMatch(Locale.GERMAN::equals))) {
+      return Response.status(Status.BAD_REQUEST).build();
```

Comment:
don't fail if you don't know that locale, just silently move back to your default, you're up to nasty bugs otherwise

> This header serves as a hint when the server cannot determine the target content language otherwise (for example, use a specific URL that depends on an explicit user decision). The server should never override an explicit user language choice. The content of Accept-Language is often out of a user's control (when traveling, for instance). 

> A user may also want to visit a page in a language different from the user interface language. The server possibly can send back a [406](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406) (Not Acceptable) error code when unable to serve content in a matching language. However, such a behavior is rarely implemented for a better user experience, and servers often ignore the Accept-Language header in such cases.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language